layouts/partials/partnerships_product_item.html

```diff
@@ -6,9 +6,9 @@
     <div class="col-sm-10 col-sm-offset-1 col-xs-12">
       <div class="title-container">
         <div>
-          <h4 class='{{ .Params.phase }}{{ if eq (string (.Site.Language)) "fr" }}fr{{end}}'>
+          <h3 class='{{ .Params.phase }}{{ if eq (string (.Site.Language)) "fr" }}fr{{end}}'>
```

Comment:
Can you check where this template is used? If it's used anywhere else we have to make sure this is still valid. (There are 2 of these templates with different headers if memory serves, maybe this allows consolidating to one?)

Reply:
It's only used once in `layouts/section/partnerships.html`. Since in that section I removed the `<h3>` tag and there is only an `<h2>` tag left, this looks like it satisfies the requirements of sequential headers.